tt_metal/impl/dispatch/command_queue_interface.hpp

```diff
@@ -548,7 +1055,7 @@ class SystemMemoryManager {
         }
 
         // All data needs to be 32B aligned
-        uint32_t push_size_16B = align(push_size_B, 32) >> 4;
+        uint32_t push_size_16B = align(push_size_B, dispatch_constants::ISSUE_Q_ALIGNMENT) >> 4;
```

Comment:
can we just use `PCIE_ALIGNMENT` rather than creating new `ISSUE_Q_ALIGNMENT`?

Reply:
Opted not to in case we change the issue Q alignment, like how completion Q alignment is it's own thing. If we don't think that's a possibility then I'll switch it over